src/notebook.cpp

```diff
@@ -249,7 +250,7 @@ QString Notebook::color() const
 void Notebook::setColor(const QString &color)
 {
     d->mModifiedDate = QDateTime::currentDateTimeUtc();
-    d->mColor = color;
+    d->mColor = color.isEmpty() ? QString::fromLatin1("#0000FF") : color;
```

Comment:
Now the 0000FF seems to get used for all the notebooks that don't set a color when earlier it was just the default notebook. Allowing and defaulting the user created notebooks to empty value might still make sense, kind of interpreted as not defined?

Reply:
Good point, I've reverted this change and I assign the default colour in sqlitestorage, on initialisation.